scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatToken.scala

```diff
@@ -43,7 +44,9 @@ object FormatToken {
     */
   case class Meta(
       between: Array[Token],
-      idx: Int
+      idx: Int,
+      lowner: Tree,
+      rowner: Tree
```

Comment:
What about `leftOwner`, `rightOwner` instead?
Personally, I prefer explicit names in Scala.

Reply:
no problem, will do.